sdk/python/feast/data_source.py

```diff
@@ -466,32 +508,56 @@ class RequestSource(DataSource):
 
     def __init__(
         self,
-        name: str,
-        schema: Union[Dict[str, ValueType], List[Field]],
+        *args,
+        name: Optional[str] = None,
+        schema: Optional[Union[Dict[str, ValueType], List[Field]]] = None,
         description: Optional[str] = "",
         tags: Optional[Dict[str, str]] = None,
         owner: Optional[str] = "",
     ):
         """Creates a RequestSource object."""
-        super().__init__(name=name, description=description, tags=tags, owner=owner)
-        if isinstance(schema, Dict):
+        positional_attributes = ["name", "schema"]
+        _name = name
+        _schema = schema
+        if args:
+            warnings.warn(
+                (
+                    "requestsource parameters should be specified as a keyword argument instead of a positional arg."
```

Comment:
similar comment. RequestSource instead of requestsource

Reply:
Done.